test/css/validate.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert').strict;
+const { definitionSyntax } = require('css-tree');
+
+const css = require('@webref/css');
+
+const cssValues = [
+  { type: 'property', prop: 'properties', value: 'value' },
+  { type: 'extended property', prop: 'properties', value: 'newValue' },
+  { type: 'descriptor', prop: 'descriptors', value: 'value' },
+  { type: 'value space', prop: 'valuespaces', value: 'value' }
+];
+
+css.parseAll().then(all => {
+  for (const [shortname, json] of Object.entries(all)) {
```

Comment:
```suggestion
  for (const [shortname, data] of Object.entries(all)) {
```

I usually try to call only strings (or bytes) JSON, and pick some other name for the parsed value.